src/main/kotlin/nl/info/zac/app/search/converter/RestZoekResultaatConverter.kt

```diff
@@ -17,18 +18,25 @@ import net.atos.zac.search.model.zoekobject.TaakZoekObject
 import net.atos.zac.search.model.zoekobject.ZaakZoekObject
 import net.atos.zac.search.model.zoekobject.ZoekObject
 import net.atos.zac.search.model.zoekobject.ZoekObjectType
+import nl.info.client.zgw.util.extractUuid
+import nl.info.client.zgw.ztc.ZtcClientService
 import nl.info.zac.app.search.model.AbstractRestZoekObject
+import nl.info.zac.app.search.model.RestZaakKoppelenZoekObject
+import nl.info.zac.app.search.model.RestZaakZoekObject
 import nl.info.zac.app.search.model.RestZoekParameters
 import nl.info.zac.app.search.model.RestZoekResultaat
 import nl.info.zac.app.search.model.toRestDocumentZoekObject
 import nl.info.zac.app.search.model.toRestTaakZoekObject
 import nl.info.zac.app.search.model.toRestZaakZoekObject
 import nl.info.zac.util.AllOpen
 import nl.info.zac.util.NoArgConstructor
+import java.util.UUID
 
 @NoArgConstructor
 @AllOpen
 class RestZoekResultaatConverter @Inject constructor(
+    private val zrcClientService: ZrcClientService,
```

Comment:
Same here. I would much rather keep such dependencies out of converter classes if we can. This kind of logic does not belong in converters I think but more in service classes.

Reply:
We can:
* pass function to `convert`, so that services are injected outside converter
```
fun convert(
        zoekResultaat: ZoekResultaat<out ZoekObject>,
        informationObjectTypeUuid: UUID,
        documentKoppelbaarFn: (zaakId: String?, informationObjectTypeUuid:UUID) -> Boolean
)
```
* extract the convert logic outside the converter to make use of ZRC/DRC services

Both seem wrong in different ways :( But higher order function approach actually solves the issue with having the logic and dependencies out of the converter

@edgarvonk wdyt?

Reply:
And of course 3rd approach would be to:
* have a loop to build the return Rest object in Rest service
* convert partially the object with the converter
* add the rest of the data in service

Last step means that the data object fields have to be defined with `var` instead of `val`

Reply:
Extracted DRCClientService usage out of converter